tempest-dynamodb-local/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import com.vanniktech.maven.publish.JavaLibrary
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+
+plugins {
+  `java-library`
+  id("com.gradleup.shadow")
+  id("com.vanniktech.maven.publish.base")
+}
+
+dependencies {
+  // Ignore transtive dependencies and insyead manage explicitly.
+  implementation(libs.awsDynamodbLocal) {
+    isTransitive = false
+  }
+
+  // Implementation dependencies will be shaded in the JAR.
+  implementation(libs.bundles.jackson)
+  implementation(libs.bundles.jetty)
+  implementation(libs.kotlinStdLib)
+
+  // Shadow dependencies will not be shaded.
+  shadow(libs.antlr4Runtime)
+  shadow(libs.aws2Dynamodb)
+  shadow(libs.aws2DynamodbEnhanced)
+  shadow(libs.aws2Pinpoint)
+  shadow(libs.awsDynamodb)
+  shadow(libs.commonsCli)
+  shadow(libs.commonsLang3)
+  shadow(libs.guava)
+  shadow(libs.slf4jApi)
+  shadow(libs.bundles.sqlite4java)
+}
+
+tasks.shadowJar {
+  // Dependencies to be shaded must be explicitly included as dependencies.
+  dependencies {
+    include(dependency("com.amazonaws:DynamoDBLocal"))
+    include(dependency("com.fasterxml.jackson.core:.*"))
+    include(dependency("com.fasterxml.jackson.dataformat:.*"))
+    include(dependency("com.fasterxml.jackson.datatype:.*"))
+    include(dependency("com.fasterxml.jackson.module:.*"))
+    include(dependency("org.eclipse.jetty:.*"))
+  }
+
+  // Relocate packages to avoid conflicts.
+  relocate("com.amazon.dynamodb.grammar", "shaded.com.amazon.dynamodb.grammar")
```

Comment:
nit: it might be better to use a more unique prefix, like `tempest.shaded` rather than `shaded`. In case somebody else is also shading these classes and some poor downstream consumer gets both copies

Reply:
Good call, added a project-specific prefix to the package name.